REQUIRE

```diff
@@ -1,14 +1,9 @@
 julia 0.5
-Requests 0.3.10
-BinDeps 0.4.4
 ProtoBuf 0.3.0
 PyCall 1.7.1
 Distributions 0.10.2
-Conda 0.2.3
 JLD 0.6.3
 FileIO 0.1.2
-MNIST 0.0.2
```

Comment:
Can we keep this one? It's used by the MNIST examples and is just a tiny dependency.